tools/Google.Cloud.Tools.GenerateDocfxSources/Program.cs

```diff
@@ -177,7 +177,13 @@ private static void CopyAndGenerateArticles(ApiMetadata api, string inputDirecto
         /// </summary>
         private static string TransformDocTemplate(ApiMetadata api, string text)
         {
-            string title = $"# {api.Id}";
+            string title = 
```

Comment:
I think it would be better to have this either as a new "tag" or as part of the description. (I'd put it below the main description rather than before it.) It doesn't make as much sense to have it as part of the "title" tag.

Reply:
But then we would have to add the new tag on all the index.mds manually right? Well I can do that it's not that much anyways.

Reply:
Not necessarily manually. I'd suggest:

- Updating the template in the project generator
- Deleting all the index.md files
- Regenerating the projects
- Checking each change carefully. Only the ones with manually-written doc files would need changing at that point.

Reply:
Oh I see, let me try that.